waku/v2/node/peer_manager/waku_peer_store.nim

```diff
@@ -20,74 +25,107 @@ type
     CanConnect,
     # Connected: actively connected to peer.
     Connected
-  
+
+  PeerOrigin* = enum
+    Unknown,
+    Discv5,
+    Static,
+    Dns
+
+  # Keeps track of the Connectedness state of a peer
   ConnectionBook* = ref object of PeerBook[Connectedness]
 
-  DisconnectBook* = ref object of PeerBook[int64] # Keeps track of when peers were disconnected in Unix timestamps
+  # Keeps track of when peers were disconnected in Unix timestamps
+  DisconnectBook* = ref object of PeerBook[int64]
```

Comment:
This clearly demonstrates that we need a common time module for timestamps. We should use the same timestamp format.

cc @jm-clius

Reply:
+1 noted.

Reply:
Could we make this now a `Timestamp` (which is currently just an alias for `int64`) to save some refactoring in future.